index.html

```diff
@@ -201,15 +224,15 @@ <h4>Listening to Events</h4>
 
             <div class="card col-6">
                 <header>List Template with Links</header>
-                <div id="exampleListen" class="deer-view" deer-id="http://devstore.rerum.io/v1/id/5c9d15afe4b0a44e13e6170c" deer-list="itemListElement" deer-template="list"></div>
+                <deer-view id="exampleListen" deer-id="http://devstore.rerum.io/v1/id/5c9d15afe4b0a44e13e6170c" deer-list="itemListElement" deer-template="list"></deer-view>
```

Comment:
This items could not draw, so there was nothing to click to test the listening area.

Reply:
If this was the same pageload as above, there may have been some script loading error that stopped this...